src/services/ParticipantProjectService.ts

```diff
@@ -81,6 +81,7 @@ const download = async ({
   projectId: number;
   projectMeta?: ProjectMeta;
   organization?: AcceleratorOrg;
+  // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
can we `return await` on line 175 or remove the `async` from the function definition? This warning here is legit.